Codeinwp/neve
# fix: primary and secondary menu alignments [closes #1266]

### Summary
Fixed primary and secondary menu alignments.


### Will affect visual aspect of the product

NO

### Test instructions


- Check alignments for both primary and secondary menu components;
- Check with both long, multi-line and short single-line menus;


Closes #1266.
